pinot-core/src/main/java/com/linkedin/pinot/core/operator/dociditerators/SizeBasedDocIdIterator.java

```diff
@@ -18,38 +18,31 @@
 import com.linkedin.pinot.core.common.BlockDocIdIterator;
 import com.linkedin.pinot.core.common.Constants;
 
+
 public final class SizeBasedDocIdIterator implements BlockDocIdIterator {
```

Comment:
add java docs explaining the behaviour, is it inclusive or exclusive

Reply:
Will add java doc into the BlockDocIdIterator for the behavior in another PR.